xcdat/regridder/xesmf.py

```diff
@@ -40,13 +40,24 @@ def __init__(
         extrap_dist_exponent: Optional[float] = None,
         extrap_num_src_pnts: Optional[int] = None,
         ignore_degenerate: bool = True,
-        **options,
+        **options: Any,
     ):
-        """Extension of ``xESMF`` regridder.
+        """
+        Extension of ``xESMF`` regridder.
+
+        This method extends ``xESMF`` by automatically constructing by ``xe.XESMFRegridder``
+        object and ensuring bounds and metadata are preserved in the output dataset.
+
+        The ``method`` argument can take any of the following values: `bilinear`, `conservative`,
+        `conservative_normed`, `patch`, `nearest_s2d`, or `nearest_d2s`. You can find a comparison
+        of the methods `here <https://xesmf.readthedocs.io/en/latest/notebooks/Compare_algorithms.html>`_.
+
+        The ``extrap_method`` argument can take any of the following values: `inverse_dist` or `nearest_s2d`.
+        This argument along with ``extrap_dist_exponent`` and ``extrap_num_src_pnts`` can be used to
+        configure how extrapolation is applied.
 
-        The ``XESMFRegridder`` extends ``xESMF`` by automatically constructing
-        the ``xe.XESMFRegridder`` object, preserving source bounds and
-        generating missing bounds.
+        The ``**options`` arguments are additional values passed to the ``xe.XESMFRegridder``
+        constructor. A description of these arguments can be found on `xESMF's documentation <https://github.com/pangeo-data/xESMF/blob/892ac87064d98d98d732ad8a79aa1682b081cdc2/xesmf/frontend.py#L702-L744>`_.
```

Comment:
```suggestion
        The ``**options`` arguments are additional values passed to the
        ``xe.XESMFRegridder`` constructor. A description of these arguments can
        be found on `xESMF's documentation <https://github.com/pangeo-data/xESMF/blob/892ac87064d98d98d732ad8a79aa1682b081cdc2/xesmf/frontend.py#L702-L744>`_.
```
Fixed docstring to fit in line limit